jbehave-support-core/docs/Rest-api.md

```diff
@@ -144,3 +144,28 @@ When [POST] request to [TEST]/[user/] is sent with data:
 | @header.Content-Type | application/json;charset=utf-8 |
 | @body                | Anything you want to send.     |
 ```
+
+#### Handling JSON data types
+All JSON data will be send as a string, unless specified otherwise. To specify a data type, use optional `type` column in your data table. These data types are supported:
+ + string
+ + boolean
+ + int
+ + double
+ + null
+
+If you leave the column blank, the data will be send as a string.
+
+Ex.:
+```
+| name      | data  | type    |
+| firstName | Mario | string  |
+| age       | 24    | int     |
+| height    | 1.56  | double  |
+| plumber   | true  | boolean |
+| powerup   |       | null    |
```

Comment:
debate type null vs NULL_COMMAND in data or both

Reply:
do we need int and double? afaik json distinguish only numbers

Reply:
I'd prefer using `{NULL}` in data column instead of `null` in type column (due to consistency )

Reply:
There is a debate, whether we want to use Json data types (number) or Java data types (int and double) https://github.com/EmbedITCZ/jbehave-support/pull/170#discussion_r316084049